agent/util-scripts/pbench-move-results

```diff
@@ -174,7 +174,7 @@ for dir in `/bin/ls -ort -d */ | awk '{print $8}' | grep -v "^tools-" | grep -v
     # contain the tarball and the md5 file (as ${tb}.tar.xz.md5.check); pass
     # that information on to pbench-make-result-tb.
     result_dir=$(basename ${dir})
-    result_tb_name=$(pbench-make-result-tb --result-dir ${result_dir} --target-dir ${tmp}/${controller} --user ${user} --prefix ${prefix} --xz-single-threaded ${xz_single_threaded})
+    result_tb_name=$(pbench-make-result-tb --result-dir ${result_dir} --target-dir ${tmp}/${controller} --user "${user}" --prefix "${prefix}" --xz-single-threaded ${xz_single_threaded})
```

Comment:
Why are the rest of the variables not quoted?

Reply:
I'll add the quotes, but they are not necessary because `pbench-move-results` will always have a value for them.  I was trying to keep the changes to the absolute minimum.

Reply:
Added.